src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -56,8 +66,8 @@ public Climber(OperatorInterface oi) {
         climber.configPeakOutputReverse(-1, TIMEOUT);
         climber.setNeutralMode(NeutralMode.Brake);
         climber.configSupplyCurrentLimit(climberLimit, TIMEOUT);
-        climber.configForwardLimitSwitchSource(LimitSwitchSource.FeedbackConnector, LimitSwitchNormal.NormallyOpen, 0);
-        climber.configReverseLimitSwitchSource(LimitSwitchSource.FeedbackConnector, LimitSwitchNormal.NormallyOpen, 0);
+        // climber.configForwardLimitSwitchSource(LimitSwitchSource.FeedbackConnector, LimitSwitchNormal.NormallyOpen, 0);
```

Comment:
Why were these removed?

Reply:
The limit switches aren't compatible with the Falcon, so we are implementing the limit switch checks ourselves.

Reply:
Oh ok